malariagen_data/anopheles.py

```diff
@@ -102,7 +103,7 @@ class AnophelesDataResource(
     AnophelesPca,
     AnophelesIgv,
     AnophelesAimData,
-    AnophelesHapData,
+    AnophelesHapFrequencyAnalysis,
```

Comment:
Suggest to leave the `AnophelesHapData` class explicitly here within the parents, and add `AnophelesHapData` further back, e.g., together with other analysis classes.

Reply:
Thanks @alimanfoo. It would make sense but when `AnophelesHapData` is added to the list, this error pops up:

```
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases AnophelesSnpData, AnophelesHapData, AnophelesSampleMetadata, AnophelesCnvData, AnophelesGenomeFeaturesData, AnophelesGenomeSequenceData, AnophelesBase
```

I have tried to figure out why but I have not been able to yet.

Reply:
Turns out `AnophelesHapData` really wants to be before `AnophelesSnpData` and `AnophelesCnvData`.

Reply:
Sort of. The only reason (as far as I can see) the order of these 3 would be important would be if the superclasses are used differently in `AnophelesHapData` and I don't see where they would be. I am not sure it is worth investigating too much given that it works this way, though.